mojo/stdlib/test/builtin/test_value.mojo

```diff
@@ -63,6 +63,12 @@ struct ConditionalTriviality[O: MutableOrigin, //, T: Movable & Copyable](
             self.add_event(EVENT_MOVE)
 
 
+struct StructInheritTriviality[T: Movable & Copyable](Movable & Copyable):
```

Comment:
LGTM in general, but maybe use `ExplicitlyCopyable` here? Not sure whether the mojo nightly has catch up the change (if not, maybe better to wait for a few days?). 

Not a blocker, also depends on @ConnorGray's plan on updating `Copyable`.